README.md

```diff
@@ -165,6 +166,20 @@ In case passing the request to MDG failed, the consuming service retrieves callb
 
 [[Back to the top]](#top)
 
+### Retrying
+
+If file-transmission fails to deliver the message to MDG it will make several attempts to redeliver it after delay.
+If it fails to deliver it within certain delivery window, failure notification callback will be sent to consuming service.
+
+Default length of delivery window is set in the service configuration (`initialBackoffAfterFailure` parameter).
+It can be customized per request by setting `requestTimeoutInSeconds` parameter within the request body.
+
+The service uses exponential backoff when performing retry attempts. Initial retry delay is defined by 
```

Comment:
I don't think we should mention the word "exponential". We should keep it a bit looser so we can change the implementation in the future. e.g. we also mentioned a linearly increasing wait period.